samples/drivers/led_ws2812/boards/adafruit_kb2040.conf

```diff
@@ -0,0 +1 @@
+CONFIG_WS2812_STRIP_RPI_PICO_PIO=y
```

Comment:
If you enable the driver by default, this will not be necessary.

Reply:
WS2812_STRIP_DRIVER is the `choice`, so this must be left.

Reply:
My bad, in that case, I would move it into the board's defconfig. @simonguinot what do you think about it?